src/core/annotation.js

```diff
@@ -752,6 +754,26 @@ var TextWidgetAnnotation = (function TextWidgetAnnotationClosure() {
   return TextWidgetAnnotation;
 })();
 
+var ButtonWidgetAnnotation = (function ButtonWidgetAnnotationClosure() {
+  function ButtonWidgetAnnotation(params) {
+    WidgetAnnotation.call(this, params);
+
+    this.data.noToggleToOff = this.hasFieldFlag(
+      AnnotationFieldFlag.NOTOGGLETOOFF);
+    this.data.radio = this.hasFieldFlag(AnnotationFieldFlag.RADIO);
+    this.data.pushbutton = this.hasFieldFlag(AnnotationFieldFlag.PUSHBUTTON);
+    var fieldValue = Util.getInheritableProperty(params.dict, 'V');
+    if (fieldValue) {
+      // Must be a Name object
```

Comment:
The condition doesn't actually agree with the comment, so change it to:
`if (isName(fieldValue)) {` instead (and remove the comment since it's then superfluous).